azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf

```diff
@@ -0,0 +1,92 @@
+variable "express-azure-functions" {
+  default = {
+    repository = {
+      organization    = "pagopa"
+      name            = "io-functions-express"
```

Comment:
```suggestion
      name            = "azure-functions-express"
```

Reply:
@pasqualedevita actually npm module is named [express-azure-functions](https://www.npmjs.com/package/@pagopa/express-azure-functions) not `azure-functions-express`.
If we want to maintain coherence between github and npm, we need to rename github repo.
cc @balanza

Reply:
yes absolutely my mistake

Reply:
no problem. renamed in [d83c497](https://github.com/pagopa/gitops/pull/109/commits/d83c497a9f54f080e863a00c95200e6765cfc36d) :)